src/main/java/com/esri/samples/scene/sync_map_and_scene_viewpoints/SyncMapAndSceneViewpoints.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2018 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+
+package com.esri.samples.scene.sync_map_and_scene_viewpoints;
+
+import java.util.ArrayList;
+
+import javafx.application.Application;
+import javafx.geometry.Orientation;
+import javafx.scene.Scene;
+import javafx.scene.control.SplitPane;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.GeoView;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+
+public class SyncMapAndSceneViewpoints extends Application {
+
+  private MapView mapView;
+  private SceneView sceneView;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+
+    // create split pane and JavaFX app scene
+    SplitPane splitPane = new SplitPane();
+    splitPane.setOrientation(Orientation.HORIZONTAL);
+    Scene fxScene = new Scene(splitPane);
+
+    // set title, size, and add JavaFX scene to stage
+    stage.setTitle("Sync Map and Scene Viewpoints");
+    stage.setWidth(1000);
+    stage.setHeight(700);
+    stage.setScene(fxScene);
+    stage.show();
+
+    // create a map and add a basemap to it
+    ArcGISMap map = new ArcGISMap();
+    map.setBasemap(Basemap.createImagery());
+
+    // create a scene and add a basemap to it
+    ArcGISScene scene = new ArcGISScene();
+    scene.setBasemap(Basemap.createImagery());
+
+    // set the map to a map view
+    mapView = new MapView();
+    mapView.setMap(map);
+
+    // set the scene to a scene view
+    sceneView = new SceneView();
+    sceneView.setArcGISScene(scene);
+
+    // add the map view and scene view to the split plane
+    splitPane.getItems().addAll(mapView, sceneView);
+
+    // add a view point changed listener to the map view
+    mapView.addViewpointChangedListener(viewpointChangedEvent -> {
```

Comment:
For one line lambdas you can remove the curly braces and semi-colon. Also, instead of passing the GeoView to the synchronize method, you could pass the event and then use `getSource` on the event to get the GeoView. 
```java
mapView.addViewpointChangedListener(viewpointChangedEvent -> synchronizeViewpoints(viewpointChangedEvent));
```

Reply:
This is really neat: didn't realise you could access the GeoView type in this way, thanks for the tip!

Reply:
If you are passing the event as the argument you should be able to do something like
```
mapView.addViewpointChangedListener(this::synchronizeViewpoints);
```